spec/rubocop/cop/rspec/capybara/specific_matcher_spec.rb

```diff
@@ -60,20 +60,72 @@
     RUBY
   end
 
+  %i[above below left_of right_of near count minimum maximum between text id
+     class style visible obscured exact exact_text normalize_ws match wait
+     filter_set focused disabled name value title type name
+     disabled].each do |attr|
+    it 'registers an offense for abstract matcher when ' \
+      "first argument is button element with #{attr}" do
+      expect_offense(<<-RUBY, attr: attr)
+        expect(page).to have_css("button[#{attr}]")
```

Comment:
I don't get it, do we check that we flag `have_css("button[with]")` as an offence? Is this a legitimate syntax?

Reply:
This cop only offense attributes that can be replaced by specific matcher. I think the `it` explanation was confusing. Improved description. How do you like it?